### PR TITLE
DOCS-3052 Adds Images into RUM Folder

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Adds an image from the RUM repo to size properly for Android and React Native doc pages.

### Motivation

DOCS-3052

### Preview

https://docs-staging.datadoghq.com/alai97/rum-documentation-link-cleanup-doc-update/real_user_monitoring/ios/
https://docs-staging.datadoghq.com/alai97/rum-documentation-link-cleanup-doc-update/real_user_monitoring/android/

### Additional Notes

I will update the branch to pull the preview so we can see all the changes incorporated across dd-sdk-ios and dd-sdk-android. Remember to change the branches back before merging this in!

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by apigirl — APPROVED** [review]

**Review by apigirl — APPROVED** [review]

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 1]
```diff
@@ -291,14 +291,14 @@
   - repo_name: dd-sdk-android
     contents:
     - action: pull-and-push-file
-      branch: master
+      branch: alai97/rum-documentation-link-cleanup
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:294]:
```suggestion
      branch: master
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 2]
```diff
@@ -291,14 +291,14 @@
...
       globs:
       - 'docs/rum_getting_started.md'
       options:
         dest_path: '/real_user_monitoring/android/'
         file_name: '_index.md'
         front_matters:
-          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/master/docs/rum_getting_started.md"]
+          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/alai97/rum-documentation-link-cleanup/docs/rum_getting_started.md"]
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:301]:
```suggestion
          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/master/docs/rum_getting_started.md"]
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 3]
```diff
@@ -309,14 +309,14 @@
               tag: "Homepage"
               text: "Explore Datadog RUM"
     - action: pull-and-push-file
-      branch: master
+      branch: alai97/rum-documentation-link-cleanup
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:312]:
```suggestion
      branch: master
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 4]
```diff
@@ -309,14 +309,14 @@
...
       globs:
       - 'docs/mobile_data_collected.md'
       options:
         dest_path: '/real_user_monitoring/android/'
         file_name: 'data_collected.md'
         front_matters:
-          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/master/docs/mobile_data_collected.md"]
+          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/alai97/rum-documentation-link-cleanup/docs/mobile_data_collected.md"]
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:319]:
```suggestion
          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/master/docs/mobile_data_collected.md"]
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 5]
```diff
@@ -327,14 +327,14 @@
               tag: "Homepage"
               text: "Explore Datadog RUM"
     - action: pull-and-push-file
-      branch: master
+      branch: alai97/rum-documentation-link-cleanup
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:330]:
```suggestion
      branch: master
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 6]
```diff
@@ -327,14 +327,14 @@
...
       globs:
       - 'docs/configure_rum_android_sdk.md'
       options:
         dest_path: '/real_user_monitoring/android/'
         file_name: 'advanced_configuration.md'
         front_matters:
-          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/master/docs/configure_rum_android_sdk.md"]
+          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/alai97/rum-documentation-link-cleanup/docs/configure_rum_android_sdk.md"]
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:337]:
```suggestion
          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/master/docs/configure_rum_android_sdk.md"]
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 7]
```diff
@@ -345,14 +345,14 @@
               tag: "Homepage"
               text: "Explore Datadog RUM"
     - action: pull-and-push-file
-      branch: master
+      branch: alai97/rum-documentation-link-cleanup
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:348]:
```suggestion
      branch: master
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 8]
```diff
@@ -345,14 +345,14 @@
...
       globs:
       - 'docs/rum_mobile_vitals.md'
       options:
         dest_path: '/real_user_monitoring/android/'
         file_name: 'mobile_vitals.md'
         front_matters:
-          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/master/docs/rum_mobile_vitals.md"]
+          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/alai97/rum-documentation-link-cleanup/docs/rum_mobile_vitals.md"]
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:355]:
```suggestion
          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/master/docs/rum_mobile_vitals.md"]
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 9]
```diff
@@ -363,14 +363,14 @@
               tag: "Blog"
               text: "Monitor Mobile Vitals in Datadog"
     - action: pull-and-push-file
-      branch: master
+      branch: alai97/rum-documentation-link-cleanup
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:366]:
```suggestion
      branch: master
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 10]
```diff
@@ -363,14 +363,14 @@
...
       globs:
       - 'docs/integrated_libraries_android.md'
       options:
         dest_path: '/real_user_monitoring/android/'
         file_name: 'integrated_libraries.md'
         front_matters:
-          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/master/docs/integrated_libraries_android.md"]
+          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/alai97/rum-documentation-link-cleanup/docs/integrated_libraries_android.md"]
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:373]:
```suggestion
          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/master/docs/integrated_libraries_android.md"]
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 11]
```diff
@@ -381,14 +381,14 @@
               tag: "Homepage"
               text: "Explore Datadog RUM"
     - action: pull-and-push-file
-      branch: master
+      branch: alai97/rum-documentation-link-cleanup
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:384]:
```suggestion
      branch: master
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 12]
```diff
@@ -551,14 +551,14 @@
               text: "Explore your services, resources, and traces"
 
     - action: pull-and-push-folder
-      branch: master
+      branch: alai97/rum-documentation-link-cleanup
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:554]:
```suggestion
      branch: master
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 13]
```diff
@@ -381,14 +381,14 @@
...
       globs:
       - 'docs/troubleshooting_android.md'
       options:
         dest_path: '/real_user_monitoring/android/'
         file_name: 'troubleshooting.md'
         front_matters:
-          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/master/docs/troubleshooting_android.md"]
+          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/alai97/rum-documentation-link-cleanup/docs/troubleshooting_android.md"]
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:391]:
```suggestion
          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/master/docs/troubleshooting_android.md"]
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 14]
```diff
@@ -551,14 +551,14 @@
...
-      branch: master
+      branch: alai97/rum-documentation-link-cleanup
       globs:
         - docs/rum_collection/*
       options:
         dest_dir: '/real_user_monitoring/ios/'
         path_to_remove: 'docs/rum_collection/'
         front_matters:
-          dependencies: [ "https://github.com/DataDog/dd-sdk-ios/blob/master/docs/rum_collection/" ]
+          dependencies: [ "https://github.com/DataDog/dd-sdk-ios/blob/alai97/rum-documentation-link-cleanup/docs/rum_collection/" ]
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:561]:
```suggestion
          dependencies: [ "https://github.com/DataDog/dd-sdk-ios/blob/master/docs/rum_collection/" ]
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 15]
```diff
@@ -574,7 +574,7 @@
     contents:
 
     - action: pull-and-push-file
-      branch: main
+      branch: alai97/rum-documentation-link-cleanup
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:577]:
```suggestion
      branch: master
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 16]
```diff
@@ -585,7 +585,7 @@
           kind: documentation
           beta: true
           description: "Collect RUM data from your React Native projects."
-          dependencies: ["https://github.com/DataDog/dd-sdk-reactnative/blob/main/README.md"]
+          dependencies: ["https://github.com/DataDog/dd-sdk-reactnative/blob/alai97/rum-documentation-link-cleanup/README.md"]
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:588]:
```suggestion
          dependencies: ["https://github.com/DataDog/dd-sdk-reactnative/blob/master/README.md"]
```

**Review by alai97 — COMMENTED** [review]
All approved, switching preview branches back to master.

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 17]
```diff
@@ -585,7 +585,7 @@
           kind: documentation
           beta: true
           description: "Collect RUM data from your React Native projects."
-          dependencies: ["https://github.com/DataDog/dd-sdk-reactnative/blob/main/README.md"]
+          dependencies: ["https://github.com/DataDog/dd-sdk-reactnative/blob/master/README.md"]
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:588]:
```suggestion
          dependencies: ["https://github.com/DataDog/dd-sdk-reactnative/blob/main/README.md"]
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 18]
```diff
@@ -574,7 +574,7 @@
     contents:
 
     - action: pull-and-push-file
-      branch: main
+      branch: master
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:577]:
```suggestion
      branch: main
```

**Review by alai97 — COMMENTED** [review]
React Native repo is main, not master.

**Pull request merged**